README.md

```diff
@@ -52,7 +52,28 @@ $wps['run']->silenceErrorsInPaths( '~^((?!/my-plugin/).)*$~', E_NOTICE | E_WARNI
 // Silence for plugins _except_ specific plugin.
 $wps['run']->silenceErrorsInPaths( '~/wp-content/plugins/(?!my-plugin)~', E_NOTICE | E_WARNING );
 ```
+### Testing a fatal error state
```

Comment:
Hm. I don't think this warrants readme space. WordPress installs are... rarely in a state when getting an error is a problem. :)